pkg/strategy/support/strategy.go

```diff
@@ -77,14 +78,68 @@ func (stop *PercentageTargetStop) GenerateOrders(market types.Market, pos *types
 	return targetOrders
 }
 
-// ResistanceStop is a kind of stop order by detecting resistance
-type ResistanceStop struct {
-	Interval      types.Interval   `json:"interval"`
-	sensitivity   fixedpoint.Value `json:"sensitivity"`
-	MinVolume     fixedpoint.Value `json:"minVolume"`
-	TakerBuyRatio fixedpoint.Value `json:"takerBuyRatio"`
+type TrailingStopTarget struct {
+	TrailingStopCallBackRatio float64 `json:"trailingStopCallBackRatio"`
```

Comment:
your type name is called trailing stop, so I think your json tag can ignore the prefix.